release-pypi-private/action.yml

```diff
@@ -110,6 +110,18 @@ runs:
   using: "composite"
   steps:
 
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "WARNING"
+        message: >
+          As the python ecosystem is moving to project metadata version 2.4,
+          we need to use a more recent version of pypa/gh-action-pypi-publish.
+          However, this action is not compatible with composite actions.
+          Therefore, the `ansys/release-pypi-private` action is destined to
+          disappear in a future version. We invite all users to define their
+          release job manually. As an example, see
+          https://actions.docs.ansys.com/version/stable/release-actions/index.html#release-pypi-template
+
```

Comment:
I don't think the private PyPI action will ever disappear in fact. This is the one used to publish to our private PyPI... which we will still need I guess because the Trusted Publisher approach doesn't work there